styles.css

```diff
@@ -0,0 +1,779 @@
+/* google fonts */
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=League+Gothic&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.wrapper{
+    width: 100%;
+    max-width: 1440px;
```

Comment:
- [ ] Adding max-width to your page creates a left and right padding please remove or update the width of your page

Reply:
I have changed it but I added max-width to keep the size of my website perfect, this is the same thing that a bootstrap container class does.

Reply:
- [ ] Yes but if u look closely  above  width 1440px there is spacing check screenshot below, this will also help you look out for other screen sizes above 1440px

![image](https://user-images.githubusercontent.com/69638013/157704888-689ee39e-74ef-420e-837e-f0652f83b1b0.png)